packages/@lwc/engine-server/src/utils/classes.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+const CLASSNAMES_SEPARATOR = /\W+/g;
+
+export function classNameToTokenList(value: string): Set<string> {
```

Comment:
from the code, i the return from `classNameToTokenList('foo-bar')` will be `['foo', 'bar']`?, i would assume is `['foo-bar']`, is my assumption correct?

Reply:
Good catch!